cmd/riscli/routers.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	pb "github.com/bio-routing/bio-rd/cmd/ris/api"
+	"github.com/bio-routing/bio-rd/util/log"
+	"github.com/urfave/cli"
+)
+
+// NewLPMCommand creates a new LPM command
+func NewGetRoutersCommand() cli.Command {
+	cmd := cli.Command{
+		Name:  "routers",
+		Usage: "get all routers and vrfs available",
+		Flags: []cli.Flag{},
+	}
+
+	cmd.Action = func(c *cli.Context) error {
+		conn := SetupGRPCClient(c)
+		defer conn.Close()
+
+		client := pb.NewRoutingInformationServiceClient(conn)
+		err := getRouters(client, c.GlobalString("router"), c.GlobalUint64("vrf_id"), c.GlobalString("vrf"))
+		if err != nil {
+			log.Errorf("Get Routers failed: %v", err)
+			os.Exit(1)
+		}
+
+		return nil
+	}
+
+	return cmd
+}
+
+func getRouters(c pb.RoutingInformationServiceClient, routerName string, vrfID uint64, vrf string) error {
+	resp, err := c.GetRouters(context.Background(), &pb.GetRoutersRequest{})
+	if err != nil {
+		return fmt.Errorf("unable to get client: %w", err)
+	}
+	for _, r := range resp.GetRouters() {
+		fmt.Printf("Router %v at %v\n", r.SysName, r.Address)
+		fmt.Printf("VRFs: %v\n\n", r.VrfIds)
```

Comment:
Why not print human readable route distinguisher? See https://pkg.go.dev/github.com/bio-routing/bio-rd@v0.1.7/routingtable/vrf#RouteDistinguisherHumanReadable